CONTRIBUTING.md

```diff
@@ -49,21 +55,24 @@ Test262 supports the following tags:
  - [**description**](#description) (required)
  - [**info**](#info)
  - [**negative**](#negative)
- - [**es5id**](#es5id)
- - [**es6id**](#es6id)
  - [**esid**](#esid) (required for new tests)
  - [**includes**](#includes)
  - [**timeout**](#timeout)
  - [**author**](#author)
  - [**flags**](#flags)
  - [**features**](#features)
 
+The following tags are deprecated, but exist in old tests:
+
+ - [**es5id**](#es5id) (deprecated)
+ - [**es6id**](#es6id) (deprecated)
+
 #### description
 **description**: [string]
 
-This is one of two required frontmatter tags. It should be a short, one-line
-description of the purpose of this testcase.  This is the string displayed by
-the browser runnner.
+This is one of two required frontmatter tags. It should be a short (less than
+100 characters), one-line description of the purpose of this testcase.  This 
+is the string displayed by the browser runnner.
```

Comment:
> This is the string displayed by the browser runnner.

We should remove this.

Maybe the 100 characters limit should be a relaxed as a convention, not a rule. What do you think?

Reply:
I think we can relax it to a "suggestion"?